### ISSUE TITLE
Zero point manual grading questions are autograded

### ISSUE THREAD

**Issue by trombonekenny** [PrairieLearn/PrairieLearn#10809]
I noticed that the `manualPoints: 0` questions on my exams show up with a Save & Grade button instead of just Save. From @jonatanschroeder on Slack:

> Zero total points currently means autograding. This should be made more flexible once manual percentages are implemented (cue a reminder for @mwest1066  to review that PR), but at this point you can't have a zero points question that is purely manual grading.

I have a few questions I don't expect the students to grade (ala #3841) but I want them to work with the element. A consequence of `points: 0` on an exam is they only get 1 submission and the question goes `editable: False`. I've worked around this in the meantime with `server.py` marking the submission invalid.

But I still wanted to file an issue to track we should be able to have Save only zero point questions.

**Comment by mwest1066** [comment]
Can we consolidate this as a comment on #3841 rather than a separate issue?

**Comment by jonatanschroeder** [comment]
For zero-point questions that are not intended to be submitted: #3841 is the place to go.

For zero-point questions that are intended to be manually graded, not autograded: once #8890 lands, we can rely on the manual percentage (regardless of points) to determine if a question is manually graded.

So I believe this is a duplicate of either of these.

**Comment by trombonekenny** [comment]
I think it is an error case to have a manually graded question present as Save & Grade to the students, even if it's worth zero points. It shouldn't be configured one way and execute a different way.

If we're tracking that resolution under #8890, then I agree it's a duplicate. If we thought that fix was long enough out, maybe consider leaving this bug in place to address with a sync error or warning?

**Comment by jonatanschroeder** [comment]
If your concern is the disconnect between grading method and how zero points are tracked, then indeed #8890 will take care of this.

**Comment by mwest1066** [comment]
Sounds like we can close this issue in favor of #8890? Or do you want to also comment on #3841, @trombonekenny?
